matlab_simulation/code v1.0/ME597-7-NonlinearSteering/angleWrap.m

```diff
@@ -0,0 +1,17 @@
+%% angleWrap
+% A function that receives an angle and binds it between [-pi, pi].
+%
+% Input:
+% angle - Any arbitray angle measurement in radians
+%
+% Output:
+% angle - The same angle except numerically bound from -pi to pi
+function [angle] = angleWrap(angle)
+
+while ( angle < -pi || angle > pi)
+    if (angle < -pi)
+        angle = angle + 2*pi;
+    else
+        angle = angle - 2*pi;
+    end
+end
```

Comment:
Modified the code into one line. The function has a description of its purpose as well as a one line solution may be a bit confusing to some.